CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender
+identity and expression, level of experience, education, socio-economic status,
+nationality, personal appearance, race, religion, or sexual identity
+and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming,
+diverse, inclusive, and healthy community.
```

Comment:
> diverse, inclusive

On the contrary, the project will I presume have bounds, and we will reject contributions that don't meet the desired quality. 

Who we might wish to attract is a strategic decision, and we shouldn't tie our hands to one approach:

For example, for inclusive, we likely will not to cater to users of other operating systems. Similarly, for diverse, while one could write the project in a variety of programming languages for the sake of attracting people who are comfortable in different langauges, we may find other considerations more important.

Reply:
Agreed. 

I would add that experimenting is a great way to improve knowledge and find better ways to solve issues. In the end "Impossible" is so until someone makes it happen...